lib/Frontend/ParseableInterfaceModuleLoader.cpp

```diff
@@ -356,48 +356,39 @@ class swift::ParseableInterfaceBuilder {
     return false;
   }
 
-  /// Populate the provided \p Deps with \c FileDependency entries including:
-  ///
-  ///    - All the dependencies mentioned by \p SubInstance's DependencyTracker,
-  ///      that were read while compiling the module.
-  ///
-  ///    - For any file in the latter set that is itself a .swiftmodule
-  ///      living in \p cachePath, all of _its_ dependencies, copied
-  ///      out to avoid having to do recursive scanning when rechecking this
-  ///      dependency in the future.
+  /// Determines if the dependency with the provided path is a swiftmodule in
+  /// either the module cache or prebuilt module cache
+  bool isCachedModule(StringRef DepName) const {
+    if (moduleCachePath.empty() && prebuiltCachePath.empty())
```

Comment:
Nitpick: I think in practice `moduleCachePath` will never be empty. I wouldn't remove the check from below just in case but we probably don't need the early exit here.